extensions/wikia/VideoHandlers/videoInfo/VideoInfo.class.php

```diff
@@ -415,6 +378,8 @@ protected static function getMemcKey( $videoTitle ) {
 	 * save to cache
 	 */
 	protected function saveToCache() {
+		$cache = [];
```

Comment:
Is this related to premium videos or just a fix?

Reply:
Just a fix, the variable was not initialised properly.